src/link/Coff.zig

```diff
@@ -1498,6 +1607,19 @@ pub fn flushModule(self: *Coff, comp: *Compilation, prog_node: *std.Progress.Nod
     sub_prog_node.activate();
     defer sub_prog_node.end();
 
+    {
+        var lazy_it = self.lazy_syms.iterator();
```

Comment:
Being part of `flushModule` in every linking backend, this will be run on every incremental update. Do lazy symbols ever get out-of-date/recycled/dead? If so, what then? Should we track that and free them somewhere? At the very least, should we perhaps mark them dirty? The difference between lazy and unnamed symbol is that the latter is called from either `updateFunc` or `updateDecl`, and therefore, they will be run only when needed unlike `updateLazySymbol`.

Reply:
The only currently implemented lazy symbol always reuses the same symbol, or at least it should be able to if I didn't implement it correctly.  Note that the llvm, C, and wasm backends already do this work for an error table in every call to `flushModule`.  Other lazy symbols at least have an associated decl that could probably used for tracking/update, but I haven't fully worked out how this would work yet.  The main reason I think that `lazy_syms` is required is that I'm not sure if there is a simple way to get the list of lazy symbols for a decl, only to get the optional decl associated with a lazy symbol.  This loop will need to be changed to only update the (single) lazy symbol that doesn't have an associated decl once there is a way for `updateDecl` to update the relevant lazy symbol.

Reply:
Mhm, makes sense. Tracking dirty state will be of particular interest for hot updates so that we limit the amount of syscalls and memory writes, but it's fine as is for now.